packages/components/src/passengerCard/PassengerCard.js

```diff
@@ -45,94 +25,112 @@ function getTitle(gender) {
-          value={insurance}
-          label="Travel Insurance"
-          style="normal"
-        />
-        <View style={styles.bagsRowWrapper}>
-          <Text type="secondary" style={styles.textPadding}>
-            Bags
+    return (
+      <Card style={styles.container}>
+        <View style={styles.containerName}>
+          <Icon name="passenger" />
+          <Text style={styles.passengerName} size="large">
+            {passenger}
           </Text>
-          <View>
-            {bags &&
-              bags.map(bag => (
-                <BagInformation
-                  key={bag.type}
-                  count={bag.count}
-                  type={bag.type}
-                />
-              ))}
+          {onActionPress && actionIconName && (
+            <ExtendedTouchable onPress={this.handleActionPress}>
+              <Icon
+                name={actionIconName}
+                color={defaultTokens.backgroundButtonPrimary}
+              />
+            </ExtendedTouchable>
+          )}
+        </View>
+
+        <View style={styles.containerTop}>
+          <PassengerCardDetail
+            value={nationality}
+            label="Nationality"
+            style="normal"
+          />
+          <PassengerCardDetail
+            value={dateOfBirth}
+            label="Date of birth"
+            style="normal"
+          />
+          <PassengerCardDetail value={id} label="ID" style="id_row_wrapper" />
+        </View>
+        <Separator />
+        <View style={styles.containerBottom}>
+          <PassengerCardDetail
+            value={insurance}
+            label="Travel Insurance"
+            style="normal"
+          />
+          <View style={styles.bagsRowWrapper}>
+            <Text type="secondary" style={styles.textPadding}>
+              Bags
+            </Text>
+            <View>
+              {bags &&
+                bags.map(bag => (
+                  <BagInformation
+                    key={bag.type}
+                    count={bag.count}
+                    type={bag.type}
+                  />
+                ))}
+            </View>
           </View>
         </View>
-      </View>
-    </Card>
-  );
+      </Card>
+    );
+  }
 }
 
-PassengerCard.defaultProps = {
-  passengerCount: 1,
-  name: '',
-  gender: 'other',
-  nationality: '',
-  dateOfBirth: '',
-  id: '',
-  insurance: '',
-  bags: null,
-};
-
 const styles = StyleSheet.create({
   container: {
-    paddingHorizontal: parseInt(defaultTokens.spaceMedium, 10),
-    ios: {
-      marginHorizontal: parseInt(defaultTokens.spaceXSmall, 10),
-    },
+    marginBottom: parseFloat(defaultTokens.spaceSmall),
   },
   containerName: {
     flexDirection: 'row',
     alignItems: 'center',
-    paddingTop: parseInt(defaultTokens.spaceSmall, 10),
-    paddingBottom: parseInt(defaultTokens.spaceMedium, 10),
+    paddingTop: parseFloat(defaultTokens.spaceSmall),
+    paddingBottom: parseFloat(defaultTokens.spaceMedium),
```

Comment:
I'm just curious, but is this change necessary? Personally I like `parseInt` slightly more and I'll not really expect padding from tokens to be float.
But we can definitely keep it this way.

Reply:
It's not necessary, you are right it's useless change, According to me it looks more readable without second argument. We can keep it as it was and it hopefully it will be changed in the future.

Reply:
I changed it back to original implementation